### DISCUSSION TITLE
new feature Toggle active tab

### DISCUSSION THREAD

**Discussion by jawharjob** [openstyles/stylus#1480] [Ideas]
Thanks for the great extension and
Can you add a feature  Toggle active for currently open tab with shortcut

I mean stop style the current page only, not all

**Comment by tophf** [comment]
It's a good idea, and it was [already suggested](https://github.com/openstyles/stylus/issues/35#issuecomment-286496323), but no one was interested enough to implement it.

**Reply by dmland** [reply]
☹️ Ahh, well … I guess I'll add an upvote to the above dead-end request, anyway. Maybe someone will want to implement it some day.

Thank you for keeping this project alive after Stylish went sour.
